draft-ietf-quic-transport.md

```diff
@@ -2594,7 +2594,7 @@ MUST attempt to process the remaining packets.
 
 Retry packets ({{packet-retry}}), Version Negotiation packets
 ({{packet-version}}), and packets with a short header ({{short-header}}) do not
-contain a Length field and so cannot be followed by other packets in the same
+contain a Length field and so MUST NOT be followed by other packets in the same
```

Comment:
I prefer cannot. It isn't an interoperability requirement that these not be followed by anything else, it is an outright impossibility.